test/integration/features/basic.feature

```diff
@@ -179,6 +179,16 @@ Feature: Basic
       """
       The 'minishift' VM is already stopped.
       """
+    When executing "minishift console" succeeds
```

Comment:
+1

Reply:
@amitkrout That's because we are returning exit code zero as it is exactly not a failure but a graceful exit.

Reply:
Then its fine, thanks for the clarification

Reply:
![screenshot from 2018-02-28 19-05-46](https://user-images.githubusercontent.com/1132451/36790366-9168aa36-1cba-11e8-8ac0-016b240f650b.png)